unit/src/MFEMMaterialTest.C

```diff
@@ -371,10 +217,12 @@ TEST_F(MFEMFunctorScalarMaterialTest, MFEMGenericFunctorMaterial_Exception)
 TEST_F(MFEMFunctorVectorMaterialTest, MFEMGenericFunctorVectorMaterial)
 {
   InputParameters coef_params = _factory.getValidParams("MFEMGenericFunctorVectorMaterial");
-  coef_params.set<std::vector<std::string>>("prop_names") = {"coef1", "coef2"};
-  coef_params.set<std::vector<MFEMVectorCoefficientName>>("prop_values") = {"func1", "func2"};
+  coef_params.set<std::vector<std::string>>("prop_names") = {
+      "coef1", "coef2", "constant1", "constant2"};
+  coef_params.set<std::vector<MFEMVectorCoefficientName>>("prop_values") = {
+      "func1", "func2", "{0.", "1.", "2.}", "{1.5}"};
```

Comment:
What happens when only a single value is supplied like with `"{1.5}"`?

Reply:
Are all three components set to 1.5 or just the x-component?

Reply:
`{1.5}` is interpreted as a single-component vector. If it gets used along with vectors with more components then the missing components of this one will be treated as 0. So, in effect, we are just setting the x-component.